a5/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# rosindustrial/a5:kinetic
+FROM rosindustrial/noether:kinetic
+MAINTAINER Austin.Deric@gmail.com
```

Comment:
I would go for a label..
The `MAINTAINER` line was added mostly to make the layers unique.